src/lib/backend.ai-client-esm.ts

```diff
@@ -697,6 +697,9 @@ class Client {
       this._features['model-service-validation'] = true;
       this._features['max-customized-image-count'] = true;
     }
+    if (this.isManagerVersionCompatibleWith('24.03.4')) {
+      this._features['extra-mounts'] = true;
```

Comment:
How about changing the key name to be more contextual, for example, `endpoint-extra-mounts`?

Reply:
Please check this part. I think that some code has been reverted unexpectedly.

Reply:
Oops I just fixed it.